aries/aries_vcx/src/common/proofs/prover/mod.rs

```diff
@@ -59,13 +59,13 @@ pub async fn generate_indy_proof(
     let proof = anoncreds
         .prover_create_proof(
             wallet,
-            proof_req_data_json,
+            serde_json::from_str(proof_req_data_json)?,
             &requested_credentials,
-            settings::DEFAULT_LINK_SECRET_ALIAS,
-            &schemas_json,
-            &credential_defs_json,
-            Some(&revoc_states_json),
+            &settings::DEFAULT_LINK_SECRET_ALIAS.to_string(),
+            serde_json::from_str(&schemas_json)?,
+            serde_json::from_str(&credential_defs_json)?,
+            Some(revoc_states_json),
```

Comment:
perhaps leave a "todo" comment to update the sources of this data to return them typed already

Reply:
I think that is not necessary as it is obviously wrong and remedy is already in progress.

Reply:
makes sense